package.json

```diff
@@ -1,55 +1,31 @@
 {
-  "name": "@readme/eslint-config",
-  "version": "8.5.1",
-  "description": "ReadMe coding standards",
-  "main": "index.js",
-  "author": "Jon Ursenbach <jon@ursenba.ch>",
-  "license": "ISC",
-  "repository": {
-    "type": "git",
-    "url": "git@github.com:readmeio/eslint-config.git"
-  },
-  "engines": {
-    "node": "^12 || ^14 || ^16"
-  },
+  "name": "standards",
+  "version": "0.0.0",
+  "private": true,
+  "description": "Core coding and style standards for all ReadMe technology.",
   "scripts": {
+    "lint": "npx lerna run lint --stream",
     "prepare": "husky install",
-    "prettier": "prettier --list-different --write \"./**/**.js\"",
-    "release": "npx conventional-changelog-cli -i CHANGELOG.md -s && git add CHANGELOG.md",
-    "test": "eslint ."
+    "publish": "npx lerna publish",
+    "test": "npx lerna run test --stream",
+    "version": "npx conventional-changelog-cli --pkg lerna.json -i CHANGELOG.md -s && git add CHANGELOG.md"
```

Comment:
I believe this should run `lerna version` instead, which will auto-generate the changelog based on conventional commits in the history since the last publish for each package.

Reply:
I believe `lerna version` will run this command when it runs

Reply:
This is the flow we've used in [api](https://github.com/readmeio/api/blob/main/package.json#L13) and [api-explorer](https://github.com/readmeio/api-explorer/blob/main/package.json#L29) fwiw

Reply:
Oh I see, sorry I missed this message before pushing up a commit.

Any opinions on whether you'd prefer to auto-generate and format the changelogs using `conventional-changelog-cli` vs the built-in one that `lerna` comes installed with? I believe `lerna` comes bootstrapped with [`standard`](https://github.com/conventional-changelog/standard-version), and I figure why not take advantage of what it comes with.

Happy to change this back!

Reply:
I don't have a preference! Happy to use the Lerna one.